payload-files/nextjs-13_2_0/BaseServer.renderToResponse--transaction.json

```diff
@@ -4,13 +4,16 @@
     "event_id": "[[ID1]]",
     "sdk": {
       "name": "sentry.javascript.nextjs",
-      "version": "7.110.1"
+      "version": "8.0.0-beta.3"
     },
     "sent_at": "[[ISODateString]]",
     "trace": {
       "environment": "qa",
       "public_key": "[[publicKey]]",
-      "trace_id": "[[ID2]]"
+      "sample_rate": "1",
+      "sampled": "true",
+      "trace_id": "[[ID2]]",
+      "transaction": "BaseServer.renderToResponse"
```

Comment:
so whyyyy is this coming in here? 🤔 we should be filtering these out, at least I thought so... @lforst could you also give this a look?

Reply:
I think Next.js didn't have the `next.span_type` attributes yet in that version 🤔 We need to find another way of filtering them.

Reply:
Opened a PR https://github.com/getsentry/sentry-javascript/pull/11722

Reply:
But the transactions are still missing 🤔 Normally, a "transaction" is saved and when an error happens, a "transaction" and "event" file is saved. In this setup the transaction is missing sometimes (see "Files Changed"). In those cases, another file is created that is based on the transaction name (which is BaseServer or similar).